tests/containers/helm.pm

```diff
@@ -14,14 +14,37 @@ use version_utils qw(is_sle);
 use registration qw(add_suseconnect_product get_addon_fullname);
 use mmapi 'get_current_job_id';
 
+
+sub install_kubectl {
+    my $version = get_var('PUBLICCLOUD_KUBECTL_VERSION', 'v1.22.2');
```

Comment:
maybe we can name it just `KUBECTL_VERSION` or `CONTAINERS_KUBECTL_VERSION` since it's a container test and has nothing to do with Public Cloud testing (even though we use Amazon in the tests).

Reply:
Done